### PR TITLE
Updated package.xml to format 2

### PR DESCRIPTION
**Pull request by hsd-dev**

### CODE REVIEW

File: industrial_moveit_benchmarking/package.xml [hunk 1]
```diff
@@ -9,19 +9,14 @@
...
-  <build_depend>pluginlib</build_depend>
-  <build_depend>constrained_ik</build_depend>
-
-  <run_depend>roscpp</run_depend>
-  <run_depend>stomp_moveit</run_depend>
-  <run_depend>moveit_core</run_depend>
-  <run_depend>moveit_ros_planning</run_depend>
-  <run_depend>pluginlib</run_depend>
-  <run_depend>constrained_ik</run_depend>
+  <depend>roscpp</depend>
```
Comment (ipa-nhg) [inline industrial_moveit_benchmarking/package.xml:12]: roscpp is three times :confused:

Comment (hsd-dev) [reply industrial_moveit_benchmarking/package.xml:12]: Fixed.

**Review by ipa-nhg — APPROVED** [review]
Thanks!

**Review by jrgnicho — APPROVED** [review]

**Comment by jrgnicho** [comment]
@ipa-hsd thanks for the contribution

**Pull request merged**